### DISCUSSION TITLE
original path of parameter-file

### DISCUSSION THREAD

**Discussion by twitnic** [jilleb/MQB-FPA#97] [Q&A]
I once read a post that said where ODIS normally stores the parameter files when you do an online synchronization.
Does anyone else know this?

**Comment by TTTAteca** [comment]
Yes, it does, until the session closes.

**Reply by twitnic** [reply]
Do you know where? On mqbtools is written:

> If you have access to ODIS Online, you can get the exact same dataset for your car. ODIS saves the datasets here during parametrisations:
> C:\Program Files (x86)\Offboard_Diagnostic_Information_System_Service\configuration\org.eclipse.osgi\bundles\114\data\dms2\

But this path does not exit. I think, this is a old path ...

**Comment by TTTAteca** [comment]
The path is fine up to and including bundles I believe, from there on, things differ, for example my dataset was in a different folder. My advise, copy paste the entire content while online with odis after programming, and then look later when time permits. You will not miss it.
